pkg/solver/controller.go

```diff
@@ -413,7 +421,7 @@ func (controller *SolverController) addJobOffer(jobOffer data.JobOffer) (*data.J
 	}
 	jobOffer.ID = id
 
-	controller.log.Info("add job offer", jobOffer)
+	controller.log.Info().Any("jobOffer", jobOffer).Msg("add job offer")
```

Comment:
Let's also add the jobOfferId as a key-value so it'll be a little easier to spot in the logs

Reply:
Yeah! Great idea. Added in 157cc4c5175bc5b951e312772925194e59acf480